lib/nosedrum/application_command.ex

```diff
@@ -95,10 +95,21 @@ defmodule Nosedrum.ApplicationCommand do
 
   alias Nostrum.Struct.{Embed, Interaction}
 
+  @typedoc """
+  A function called by `Nosedrum.Storage.followup/2` after deferring an interaction response.
+  """
```

Comment:
Should we move this section:

>   The callback should return a response similar to `c:command/1`, excluding the `:type`, `:tts?`, and `:ephemeral?`

from the later docs into the docs for this callback?